apps/studio/src/routes/signin/index.tsx

```diff
@@ -14,98 +15,284 @@ enum SignInMethod {
     GOOGLE = 'google',
 }
 
+enum SignInStep {
```

Comment:
This login screen only appears when you're not logged in so I'm gonna have to move everything you have here into a new route.